public/static/partials/user_host_options.html

```diff
@@ -64,6 +64,13 @@
         <input type="text" id="input-key-name" name="keyname" ng-model="selectedKey.name" ng-disabled="!saveKey"; placeholder="mykey.pub" key-name-unique></input>
       </p>
     </div>
+    <div style="padding-bottom:5px;" ng-show="selectedDistro.mount_allowed">
+      <input type="checkbox" ng-model="$parent.attach_volume"> Mount a volume on user's home directory </input>
```

Comment:
omit the words "on user's home directory" because we want a layer of indirection between the home directory and the volume.

Reply:
done